
controller_frequency: 10.0
default_task_wrapper: generic

plugins:
  TaskToTaskWorkflow:
    dubins: dubins_curves/Dubins
  TaskToTwistWorkflow:
    executive: project11_navigation/ExecuteTask
    transit: project11_navigation/TaskToTwistStack
    hover: hover/Hover
    path_follower: path_follower/PathFollower
    survey_line: project11_navigation/TaskToTwistStack
  TaskListToTaskListWorkflow:
    task_connector: project11_navigation/TaskConnector
  TaskListToTwistWorkflow:
    task_manager: project11_navigation/TaskManager
  TaskWrapper:
    generic: project11_navigation/GenericTask
    hover: project11_navigation/HoverTask
    survey_area: project11_navigation/SurveyAreaTask
    survey_line: project11_navigation/SurveyLineTask
    transit: project11_navigation/TransitTask

TaskToTwistWorkflow/executive:
  handlers:
    foo: bar

TaskToTwistWorkflow/transit:
  steps:
    - dubins
  last_step: path_follower

TaskToTwistWorkflow/survey_line:
  last_step: path_follower
